src/Server.zig

```diff
@@ -1327,6 +1327,15 @@ pub fn declToCompletion(context: DeclToCompletionContext, decl_handle: analysis.
     var allocator = context.server.arena.allocator();
 
     const tree = decl_handle.handle.tree;
+    const name = tree.tokenSlice(decl_handle.nameToken());
+
+    const is_cimport = std.mem.eql(u8, std.fs.path.basename(decl_handle.handle.uri), "cimport.zig");
+    if (is_cimport) {
+        if (std.mem.startsWith(u8, name, "_")) return;
+        if (std.mem.eql(u8, name, "linux")) return;
+        if (std.mem.eql(u8, name, "unix")) return;
```

Comment:
There are other symbols like these for windows:
```zig
pub const WIN32 = @as(c_int, 1);
pub const WINNT = @as(c_int, 1);
pub const WIN64 = @as(c_int, 1);
```

Looks like there are none without an underscore at the beginning on macos, but there might be more symbols like this on other targets.

Reply:
One way to solve this could be to translate-c an empty file for the target, and extract the identifiers that are present by default that way.

Reply:
> One way to solve this could be to translate-c an empty file for the target, and extract the identifiers that are present by default that way.

Definitely doable, but I don't really think its worth the effort right now for such a small quality of life improvement. I will add a TODO on this idea.